src/dev/impl/DevToys/ViewModels/Tools/Converters/JsonYaml/DoubleConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿#nullable enable
+
+using System;
+using YamlDotNet.Core;
+using YamlDotNet.Core.Events;
+using YamlDotNet.Serialization;
+
+namespace DevToys.ViewModels.Tools.JsonYaml
+{
+    internal class DoubleConverter : IYamlTypeConverter
+    {
+        public bool Accepts(Type type)
+        {
+            return type == typeof(double);
+        }
+
+        public object ReadYaml(IParser parser, Type type)
+        {
+            throw new NotImplementedException();
```

Comment:
What happens if the user converts from YAML to JSON? Would this method be called?